db-service/test/cqn4sql/model/update.cds

```diff
@@ -0,0 +1,23 @@
+// dont use virtual key `isActiveEntity` in `UPDATE … where (<key>) in <subquery>`
+// in case of path expressions
+namespace bookshop;
+
+entity Books {
+  key ID : Integer;
+  title  : String;
+  stock  : Integer;
+  author : Association to Authors;
+}
+
+entity Authors {
+  key ID : Integer;
+  name   : String;
+  alive  : Boolean;
+}
+
+service CatalogService {
+    @odata.draft.enabled
+    @readonly entity Books as projection on bookshop.Books;
```

Comment:
```suggestion
    entity Books as projection on bookshop.Books;
```

Not relevant for this test, but it's cleaner (readonly draft-enabled entities don't make much sense)